src/router.js

```diff
@@ -1,12 +1,24 @@
 import { h, createContext, cloneElement, toChildArray } from 'preact';
-import { useContext, useMemo, useReducer, useLayoutEffect, useRef } from 'preact/hooks';
+import {
+	useContext,
+	useMemo,
+	useReducer,
+	useLayoutEffect,
+	useRef
+} from 'preact/hooks';
```

Comment:
Please revert all formatting-related changes.

The only lines changed should be those relevant to the issue at hand. Helps keep the blame sane over time.

Reply:
Fixed

Reply:
Not quite, majority of the diff is still formatting changes.

Reply:
Okay, I see. Can I add the Prettier configuration here? Although I have installed the Prettier VSCode plugin, I still get the default format when formatting without configuring it. Maybe it's in another PR?

https://github.com/preactjs/wmr/blob/3c5672ecd2f958c8eaf372d33c084dc69228ae3f/package.json#L61-L67

Reply:
You can disable your 'format on save' setting or selectively stage changes.

This is something that should be addressed locally in how you use your tools, not in this repo, IMO.

Reply:
I think not, as each project may require different formatting configurations, for example, [wmr](https://github.com/preactjs/wmr/blob/3c5672ecd2f958c8eaf372d33c084dc69228ae3f/package.json#L61) and [preact-router](https://github.com/preactjs/preact-router/blob/93f1cda38c1afe36ca643914d560544d3c9f44bd/package.json#L75), but VSCode has only one configuration file. I guess this is also why these two projects have explicitly specified the formatting configuration within the project?

Reply:
Sorry, I don't mean to be rude but this is definitely a situation for you to use (or learn) one of the above mentioned methods.

Running prettier over every file you change is not something projects should adjust for.